Source/monster.cpp

```diff
@@ -3540,6 +3540,26 @@ void (*AiProc[])(int i) = {
 	/*AI_BONEDEMON*/ &BoneDemonAi
 };
 
+bool DirOK(const Monster &monster, Point position, Direction mdir)

```

Comment:
Reuse of the `DirOK` name is a bit confusing to me :)

```suggestion
bool IsRelativeMoveOK(const Monster &monster, Point position, Direction mdir)
```

Reply:
Renamed 🙂